internal/cluster/nodes/picker_consistent.go

```diff
@@ -0,0 +1,46 @@
+package nodes
+
+import (
+	"context"
+
+	"stathat.com/c/consistent"
```

Comment:
Any reason to not use `hash/fnv`? Or does this do something extra I am missing?

Reply:
It does consistent hashing, so ideally when a node comes up or goes down, most things stay on the same node, and only a small amount of data moves around.  During a re-convergence, we want as much stuff to remain on the same host as possible, because we only get one forward.

Anything that's already been forwarded, and ends up on the wrong host due to the re-convergence, gets dropped.